### ISSUE TITLE
NUllsafeMethodCall does not exist with v10 compatibility after update to 10.4.6

### ISSUE THREAD

**Issue by X-Tender** [FriendsOfTYPO3/extension_builder#313]
Get this error when try to access the EB backend Module

```
(1/2) #1278450972 TYPO3\CMS\Extbase\Reflection\Exception\UnknownClassException
Class PhpParser\Node\Expr\NUllsafeMethodCall does not exist. Reflection failed.
```

I donwloaded the branch form github and enabled it in the Extensions Module.

[extensionbuilderstack.html.zip](https://github.com/FriendsOfTYPO3/extension_builder/files/5049702/extensionbuilderstack.html.zip)

**Comment by liayn** [comment]
What catches my attention is the "NUllsafeMethodCall" name. Mind the first two chars are uppercase, which is most likely a typo somewhere. I just can't find that typo within EB or the Core at the moment.

**Comment by liayn** [comment]
You are using Core v10.4.6, right?
This is not a composer-installation and you downloaded the source of extension_builder from here, from the "v10-compatibility" branch?

**Comment by X-Tender** [comment]
Yes exactly. It worked with the 10.4.5, I know it because I kickstarted my sitepackages with it. Today I made an update and then this happened-

**Comment by X-Tender** [comment]
I searched the whole vendor folder and found this
![image](https://user-images.githubusercontent.com/1413979/89764747-4b191880-daf5-11ea-8666-28e98dd4dcd9.png)

**Comment by X-Tender** [comment]
Ok looks like it was pushed yesterday with the error /o\
https://github.com/nikic/PHP-Parser/commit/23d9c17770b5f8822315e21041c5aa834e2d612c#diff-3303ec03273f387403851a0cd5cf94e2 Also the CI failed on it.

**Comment by liayn** [comment]
So you run a composer installation. Because the Core does not ship this

**Comment by X-Tender** [comment]
Well typo3 is a composer install but not the EB.

But I think I know what happened. The first problem I was after using the new EB is taht I got this error: 
`Class 'PhpParser\ParserFactory' not found`

So I installed it via composer require.
Today I made a composer update which also pulled the new (buggy version from yesterday) and then the NUll error appeared. So now I removed and re-required the nikic/php-parser bit with version 4.5 and now it works.

So at the end its a bug of the php-parser?
